src/java.base/share/classes/sun/security/rsa/RSACore.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This copyright date should not change; you need to change the one below on line 27.

Reply:
Fixed that.